src/software/ai/hl/stp/tactic/tactic.h

```diff
@@ -7,6 +7,14 @@
 #include "software/ai/intent/intent.h"
 #include "software/ai/world/world.h"
 
+// We forward-declare the TacticVisitor interface (pure virtual class) because we need
+// to know about the existence of this class in order to accept visitors with the
+// accept() function. We cannot use an #include statement because this creates a cyclic
+// dependency
+//
+// This class can be found in ai/hl/stp/tactic/tactic_visitor.h
+class TacticVisitor;
```

Comment:
I think we can just directly include it here, we break the cycle by forward-declaring it in `tactic_visitor.h`

Reply:
no we can't:
.-> //software/ai/hl/stp/tactic:tactic
|   //software/ai/hl/stp/tactic:tactic_visitor
`-- //software/ai/hl/stp/tactic:tactic